### DISCUSSION TITLE
Start11+ explorerpatcher " Uninstall APP_XXX button ??BUG??"

### DISCUSSION THREAD

**Discussion by darknessblade** [valinet/ExplorerPatcher#922] [General]
Regarding start11 "support" with Explorerpatcher

From my testing **{IN A 100% CLEAN VM}** start11 works fine with Explorerpatcher [latest build]. 
_With 00% clean, I mean: void of any other installed programs other than Start11 and ExplorerPatcher_

but I did notice 1 Issue 
 You can uninstall nearly every app. including the Windows defender app, which causes the Defender menu to be broken

For the issue the solution would be easy:
-A feature/option to Hide the "Uninstall" APP_XXX button.
___________________
Please note, I am not sure if it caused by Start11 or from Explorerpatcher
As when i remove start11 the defender icon does not have the Uninstall button.

**Comment by valinet** [comment]
If you are talking about the "Uninstall" button being available for any app in the Start11 menu, than how is that related at all to ExplorerPatcher? better is to report the problem directly to the Start11 developers, as I can't do much about it from over here.

**Reply by darknessblade** [reply]
The reason i Reported it here, is because i am not sure whenever the problem is caused by Explorerpatcher, or start11.

Will report it later on the Start11 forum as well.
